src/main/java/be/seeseemelk/mockbukkit/ServerMock.java

```diff
@@ -907,6 +907,29 @@ public boolean getAllowNether()
 		throw new UnimplementedOperationException();
 	}
 
+	@NotNull
+	@Override
+	public String getResourcePack() {
+		return null;
+	}
+
+	@NotNull
+	@Override
+	public String getResourcePackHash() {
+		return null;
+	}
+
+	@NotNull
+	@Override
+	public String getResourcePackPrompt() {
+		return null;
+	}
+
+	@Override
+	public boolean isResourcePackRequired() {
+		return false;
+	}
```

Comment:
Invalid implementations, please follow our specifications and throw an `UnimplementedOperationException` for non-implemented method stubs.